aviary/utils/preprocessors.py

```diff
@@ -162,6 +163,120 @@ def preprocess_crewpayload(aviary_options: AviaryValues):
         raise om.AnalysisError(
             f"ERROR: In preprocesssors.py: NUM_PASSENGERS ({aviary_options.get_val(Aircraft.CrewPayload.NUM_PASSENGERS)}) is larger than the number of seats set by Design.NUM_PASSENGERS ({aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_PASSENGERS)}) .")
 
+    # Check and process cargo variables from FLOPS
+    if Settings.MASS_METHOD in aviary_options:
+        mass_method = aviary_options.get_val(Settings.MASS_METHOD)
+    else:
+        print('ERROR! mass_method not specified cannot preprocess cargo inputs')
+
+    if mass_method == LegacyCode.GASP:
+        input_cargo = Aircraft.CrewPayload.CARGO_MASS in aviary_options
+        input_max_cargo = Aircraft.CrewPayload.MAX_CARGO_MASS in aviary_options
+        input_des_cargo = Aircraft.CrewPayload.Design.CARGO_MASS in aviary_options
+
+        if Settings.PROBLEM_TYPE in aviary_options:
+            problem_type = aviary_options.get_val(Settings.PROBLEM_TYPE)
+        else:
+            problem_type = ProblemType.SIZING
+
+        if input_cargo:
```

Comment:
What would the value of input_cargo be if they don't set it? This expression will be true for both 0.0 or None, and I think you might want to be explicit about which one you want, unless the behavior is the same for unspecified, and specified at 0. This goes for all of the other conditionals as well.